opencog/tools/ide/Dockerfile

```diff
@@ -14,7 +14,7 @@ MAINTAINER Alex van der Peet "alex.van.der.peet@gmail.com"
 MAINTAINER David Hart "dhart@opencog.org"
 
 # Install non-system project dependencies
-RUN  /tmp/octool -al
+RUN  /tmp/octool -al ; ccache -C
```

Comment:
I see. It's the cache created when building the docker image. Yeah we don't need it.